wasp/apps/settings.py

```diff
@@ -80,4 +81,10 @@ def _update(self):
             else:
                 say = "Silent"
             self._nfy_slider.update()
+        settings_length = len(self._settings)
+        norm_idx = self._sett_index % settings_length
+        self._scroll_indicator.up = norm_idx > 0 and settings_length > 1
+        self._scroll_indicator.down = ( norm_idx < settings_length - 1
+                                        and settings_length > 1)
```

Comment:
This is be over-engineered. The settings app is a loop of pages rather than a list. Thus you can always scroll both up and down making it better just to lose this code.

Reply:
sure, will update in a few minutes.

Reply:
done!